auth-web/src/components/auth/staff/continuation-in/StaffContinuationInTable.vue

```diff
@@ -0,0 +1,310 @@
+<template>
+  <div>
```

Comment:
For testing purposes, it's nice to give this component an id here (or a classname if this component may have several instances).

Reply:
Updated

Reply:
Thanks. Please add an id to the Continuation Applications component, too.

Reply:
Done